tests/manager/openAccount/managerCanOpenAccount.spec.js

```diff
@@ -28,4 +49,17 @@ Test:
 Tips:
  1. Do not rely on the customer row id for the step 13. Use the ".last()" locator to get the last row.
 */
-});
\ No newline at end of file
+
+  openAccountPage = new OpenAccountPage(page);
+  bankManagerMainPage = new BankManagerMainPage(page);
+  customersListPage = new CustomersListPage(page);
+
+  await bankManagerMainPage.clickOpenAccountButton();
+  await openAccountPage.selectCustomerFromDropdown(firstName + ' ' + lastName);
```

Comment:
Ensure that the `selectCustomerFromDropdown` method correctly handles the full name format used in the dropdown. Verify that the concatenation of `firstName` and `lastName` matches the format expected by the dropdown options.